README.md

```diff
@@ -55,27 +59,38 @@ And here are links of other **HugeGraph** component/repositories:
 3. [hugegraph-commons](https://github.com/apache/incubator-hugegraph-commons) (**common & rpc** module)
 4. [hugegraph-website](https://github.com/apache/incubator-hugegraph-doc) (**doc & website** code)
 
-## Contributing
+## 🤝 Contributing
 
 Welcome to contribute to HugeGraph, please see [`How to Contribute`](CONTRIBUTING.md) for more information.
 
-## License
+Thank you to all the people who already contributed to HugeGraph!
 
-HugeGraph is licensed under Apache 2.0 License.
+[![contributors graph](https://contrib.rocks/image?repo=apache/hugegraph)](https://github.com/apache/incubator-hugegraph/graphs/contributors)
 
-## Thanks
+## ⏰ Contributor Over Time
+
+[![Stargazers Over Time](https://contributor-overtime-api.git-contributor.com/contributors-svg?chart=contributorOverTime&repo=apache/incubator-hugegraph)](https://git-contributor.com?chart=contributorOverTime&repo=apache/incubator-hugegraph)
+
+## 👍 Stargazers Over Time
+
+![Stargazers over time](https://api.star-history.com/svg?repos=apache/incubator-hugegraph&type=Date)
```

Comment:
Thanks, consider add the 2 charts in website/doc repo first? (statistic info could enhance **later**)

Reply:
I create a [new discussion](https://github.com/apache/incubator-hugegraph/discussions/2229) for this question.We can discuss this question in this discussion with the community.

Reply:
@imbajin  I delete these two controversial charts. After the community has conclusions, you can add it as according to the discussion. Do you think this is okay?